modules/interacting-serverless-apps-http2-gRPC-up-to-4-9.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// serverless/knative-serving/external-ingress-routing/using-http2-gRPC.adoc
+
+:_content-type: PROCEDURE
+[id="interacting-serverless-apps-http2-gRPC-up-to-4-9_{context}"]
+= Interacting with a serverless application using HTTP2 and gRPC in {product-title} 4.9 and older
+
+[IMPORTANT]
+====
+This method needs to expose Kourier Gateway using the `LoadBalancer` service type. You can configure this by adding the following YAML to your `KnativeServing` custom resource definition (CRD):
+
+[source,yaml]
+----
+...
+spec:
+  ingress:
+    kourier:
+      service-type: LoadBalancer
+...
+----
+====
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* Install the OpenShift CLI (`oc`).
+* You have created a Knative service.
```

Comment:
Normally I wouldn't point this out, but since you have to fix the xref above, would you mind mind making these tenses more consistent in your prerequisite sections? There are currently three tenses being use: 

* Present tense 
* Imperative mood 
* Present perfect 

I think maybe using present perfect is the best, eg 

* YOu have installed {ServerlessOperatorName} and Knative Serving d on your cluster.
* You have installed the OpenShift CLI
* You have created a Knative service.